Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -2124,7 +2124,7 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
     void clearServiceWorkerEntitlementOverride(CompletionHandler<void()>&&);
         
 #if PLATFORM(COCOA)
-    void grantAccessToCurrentPasteboardData(const String& pasteboardName);
+    void grantAccessToCurrentPasteboardData(const String& pasteboardName, std::optional<WebCore::FrameIdentifier>);
```

Comment:
Can we set default value for the frameID, so that we only need to update some callers that need to pass in a frameID?
`void grantAccessToCurrentPasteboardData(const String& pasteboardName, std::optional<WebCore::FrameIdentifier> = std::nullopt);`